dojo-starter/src/systems/actions.cairo

```diff
@@ -19,9 +20,13 @@ mod actions {
             let player = get_caller_address();
             // create a new object of Game and get the unique Id of the Game and return
             let game_id = world.uuid();
-            let new_game : Game = Game{id: game_id, player: player, score: 0}; 
+            let new_game : Game = Game{id: game_id, player: player, score: 0, state: GameStage::InProgress}; 
             return new_game;
         }
+        fn end_game(world: IWorldDispatcher, id: u32) {
+            let mut game = get!(world, id, (Game))
+            game::state = GameStage::Over;
```

Comment:
you have to set it to the world again, you get the game, you change the state and then you set it again

Reply:
done